app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -10,6 +10,14 @@ HTMLDocument.prototype.ready = new Promise((resolve) => {
 
 // If localStorage.getItem('shouldRedirectToOnboarding') is not set, i.e. null, that means we should redirect.
 function redirectableLocation() {
+  return (
+    window.location.pathname !== '/onboarding' &&
+    window.location.pathname !== '/signout_confirm' &&
+    window.location.pathname !== '/privacy'
+  );
+}
+
+function redirectableCreatorOnboardingLocation() {
```

Comment:
Refactor (blocking): Since this is being refactored, use `Array.prototype.includes()` here as well.